### ISSUE TITLE
[FR] A keyboard shortcut to switch between media fields

### ISSUE THREAD

**Issue by oguz-hanoglu** [voxel51/fiftyone#2307]
### Proposal Summary

A keyboard shortcut to switch between media fields(especially in expanded view). For example, 0 opens the default("filepath") media, 1 opens the other media.

### Motivation

-   What is the use case for this feature?

When I use Multiple Media Fields, I like to focus on specific part of an image as I switch between different media fields. This use case is similar to using a heatmap(like GradCam output) as another media field(instead of Heatmap label). So as I focus on a part, I need to switch the media field so that I can see how that specific part looks like in the other media.

-   Why is this use case valuable to support for FiftyOne users in general?

This is directly related to efficient use of Multiple Media Fields feature.

-   Why is it currently difficult to achieve this use case?

The problem is sometimes the part I like to focus is covered by the "Display Options" panel. I have to keep that panel on for quickly switching between the media fields. However, it is not possible to see that (right top) part of the image if I keep that tall panel on. (Note also that the panel with Multiple Media Fields got taller with v18)

### What areas of FiftyOne does this feature affect?

-   [x] App: FiftyOne application
-   [ ] Core: Core `fiftyone` Python library
-   [ ] Server: FiftyOne server

### Details

- A workaround solution (although not a good solution) would be zooming out in such cases, but unfortunately zoom level does not persist. (Zoom level persistence is addressed in [another issue](https://github.com/voxel51/fiftyone/issues/2162))
- Other settings in the "Display Options" are rarely used again once set, but Multiple Media Fields are frequently used. So would Multiple Media Fields options be placed on the sidebar?
-  A shortcut key would work quite well to address this problem.

**Comment by brimoor** [comment]
Nice idea! Just calling out one underlying motivation: in your opinion (or at least your use case) switching between multiple media fields is more commonly used than the other options in that menu, so it deserves a more convenient way to use it.

For videos, the `0-9` keys already have meaning (`seek to 10n% of video`), so we'd need to do some thinking on what shortcuts to use. Since there can be any number of media fields, perhaps a single key to toggle cyclically through all media fields would be nice. A natural key would be `m`, but that already has meaning for videos too:

<img width="706" alt="Screen Shot 2022-11-15 at 9 14 28 AM" src="https://user-images.githubusercontent.com/25985824/201941566-b541406b-b70c-48d7-8fc1-bec9911838ac.png">

**Comment by oguz-hanoglu** [comment]
Right. That(motivation) was mentioned in details. Now, I moved it to Motivation section. Thanks.

I do not know if different set of shortcuts is feasible. For video samples 0-9 can mean "seek to", and for image samples "multi media field switch"? Even then the question of "what if more than 10 media fields exist" remains.

If only one key would be allowed for this purpose, in my use case "go to previous media field" (hence the key 'p') would be more useful than "toggle cyclically". I keep some media fields for additional analysis when needed and they are rarely used. More generally, I go back and forth between "filepath" and another field. So in my case, I can click "p" successively to achieve this.

Also note that I propose two alternatives for addressing this problem in "Details" part. Specifically "making zoom persist when another media field is selected" would be a workaround if you do not like "yet another shortcut" solution.
